tests/pipeline/setup_hypervizor.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
back in the days, i had minimal testing of kvm in travis, which didn't require setting up the hypervisor at all. is it somehow needed now?

Reply:
Well, took this up from libvirt upstream and their testing. Was functioning well also in my godon work so far. I know it has a oddish appearence, but what else would you propose? The travis machines seemingly have no native systemd involvement there. TBH, have not revised that for quite sime time.